scout/build/disease.py

```diff
@@ -5,7 +5,7 @@
 LOG = logging.getLogger(__name__)
 
 
-def build_disease_term( disease_info: dict, alias_genes: dict = {}) -> dict:
+def build_disease_term(disease_info: dict, alias_genes: dict = {}) -> dict:
```

Comment:
Something with `black` not quite ok? The latest was a couple of weeks ago, so should have propagated?